compiler/ilgen/IlValue.cpp

```diff
@@ -0,0 +1,109 @@
+/*******************************************************************************
+ *
+ * (c) Copyright IBM Corp. 2017, 2017
```

Comment:
Is this code new or copied from other files? Just curious if the initial copyright is correct